statick_tool/plugins/reporting/json_reporting_plugin.py

```diff
@@ -90,15 +90,10 @@ def write_output(self, package: Package, level: str, line: str) -> bool:
         if not os.path.isdir(output_dir):
             logging.error("Unable to create output directory at %s!", output_dir)
             return False
-        output_file = os.path.join(output_dir, package.name + "-" + level + ".json")
-        next_output_file = os.path.join(
+
+        output_file = os.path.join(
             output_dir, package.name + "-" + level + ".statick.json"
         )
-        logging.warning(
-            "Output filename will change in statick v0.9.0 from %s to %s",
```

Comment:
should add in the changelog that this has taken place.

Reply:
You're right. I have a separate PR for a release (#474). I couldn't figure out how to have this go into the CHANGELOG until after that release. Or, we can choose to make this PR part of that release, in which case I can put this into the CHANGELOG, merge this PR, then merge that into #474. Any preferences?

Reply:
i think i'm ok either way, but since the other PRs are open, i think we should try to get them merged first.